pytorch_lightning/utilities/hparams_mixin.py

```diff
@@ -30,7 +30,8 @@ def save_hyperparameters(
         self,
         *args,
         ignore: Optional[Union[Sequence[str], str]] = None,
-        frame: Optional[types.FrameType] = None
+        frame: Optional[types.FrameType] = None,
+        logger: bool = True
```

Comment:
similarly why does the logger need to be added here?

Reply:
>similarly why does the logger need to be added here? should we add a separate property like should_log_hyperparams to decouple this?

Tbh, I like the interface where Users could define in `save_hyperparameters` itself if they wish to log the hyperparameters or not.

Reply:
Yes, IMO I think this is quite a neat API.